specification/batch/data-plane/Microsoft.Batch/stable/2018-03-01.6.1/BatchService.json

```diff
@@ -11546,7 +11546,7 @@
             "$ref": "#/definitions/ResourceFile"
           },
           "title": "A list of files that the Batch service will download to the compute node before running the command line.",
-          "description": "Files listed under this element are located in the task's working directory."
+          "description": "Files listed under this element are located in the task's working directory. There is a maximum size for the list of resource files.  When this is exceeded, the task cannot be added and the response error code will be RequestEntityTooLarge. If this occurs the recommended solution is to use an ApplicationPackage."
```

Comment:
I'm not sure the official docs should use assertive language suggesting AppPackages.  They are in the list but have their own limits (10) and scale targets (ie: blob).  Can we use try making suggestions and backing logic:

If this occurs, the collection of ResourceFiles must be reduced in size.  Alternatives include using .zip files standalone or via the Application Package feature.

Reply:
oh... and docker containers... i'm not sure we want to enumerate features because the enumeration will fall behind in the fulleness of time.  That having been said, it doesn't feel right to have AP and not DC mentioned... thoughts?

Reply:
I feel like it will be important to have as many options available as possible as we want users not to feel like they have to do one option. That being said we want to avoid information overload where before the user  knew their resourcefiles implementation worked, but now that they have exceeded some magic threshold it isn't and they have to migrate to another form of doing things.

I feel like .zip files is an important first one as it is something the user can do without any additional learning about the azure/batch infrastructure.  I feel like it would not hurt to add docker containers as the thirds option. Looking over the initial thread it looks like a lot of the NZ guys are recommending their solutions which involve using azcopy/mounting drives/start tasks, but all of these seem to need instructions to explain how to implement therefore I feel they would not belong here.

Overall it would be acceptable to point to .zip, AP, and DC as possible solutions to the problem